addon/authenticators/jwt.js

```diff
@@ -168,7 +169,7 @@ export default TokenAuthenticator.extend({
         Ember.run(function() {
           var token = response[_this.tokenPropertyName],
             tokenData = _this.getTokenData(token),
-            expiresAt = tokenData[_this.tokenExpireName],
+            expiresAt = _this.getCurrentTime() + tokenData[_this.tokenExpireName] * _this.timeFactor,
```

Comment:
this is still wrong, right @RuslanZavacky ? As discussed, the token will contain the expiration time, not how long from now it expires.